src/components/select/select.js

```diff
@@ -135,6 +137,18 @@ angular.module('material.components.select', [
  *     </md-select>
  *   </md-input-container>
  * </hljs>
+ * Multiselect with a custom separator
+ *
+ * When a developer needs to add a seperator with multi select, adding to the attribute `selection-separator-char` or
+ `selection-separator-spacing` with options. The example below demonstrate a new line for after each selected item.
+ *
+ * <hljs lang="html">
+ *   <md-input-container>
+ *     <md-select ng-model="someModel" selection-separator-char="none" selection-separator-spacing="break">
+ *       <md-option ng-value="opt" ng-repeat="opt in neighborhoods2">{{ opt }}</md-option>
+ *     </md-select>
+ *   </md-input-container>
+ * </hljs>
```

Comment:
We should add another example that uses `selection-separator-char="semicolon"` and `selection-separator-spacing="after"`.